src/reporting/ReportingToken.sol

```diff
@@ -0,0 +1,150 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/libraries/DelegationTarget.sol';
+import 'ROOT/libraries/Typed.sol';
+import 'ROOT/libraries/Initializable.sol';
+import 'ROOT/libraries/token/VariableSupplyToken.sol';
+import 'ROOT/reporting/Interfaces.sol';
+
+
+contract ReportingToken is DelegationTarget, Typed, Initializable, VariableSupplyToken, IReportingToken {
```

Comment:
It appears that you both rebased onto and merged in the reporting branch.  Easiest solution is likely to do an interactive rebase on develop and cut out the merge commit (3a260cb123af0792a09e5d320db0b7d8319afea3).  I can help walk you through the process if you haven't done this before.